### PR TITLE
[129] [I4BK7C] [I3WK54] Fix Alter Table Rename Column Bug

### LINKED ISSUE #129: Alter Table Rename Column Bug
### Software Environment:
- OpenLooKeng version (source or binary):1.4.0

- OS platform & distribution (eg., Linux Ubuntu 16.04):

- Java version:1.8
### Describe the current behavior
In OLK, After renaming a column of a table, getting NULL values as a result for that particular column.

### Describe the expected behavior
In OLK, After renaming a column of a table, correct values to be given.

### Steps to reproduce the issue
1.create table fileformat1.t6(name string, dob date) partitioned by (score int) stored as textfile;
2.insert into fileformat1.t6 values ('name1', '2001-01-01', 100),('name2', '2002-02-02',200),('name3', '2003-03-03', 100),('name4', '2005-05-06',200);
3.alter table fileformat1.t6 rename column name to alias;
4.select * from fileformat1.t6;

### Special notes for this issue

### PR DESCRIPTION
**Pull request by sumanth43**
### What type of PR is this?

kind bug 

### Which issue(s) this PR fixes:

Fixes #129  #I4BK7C #I3WK54

### Special notes for your reviewers:

### CODE REVIEW

**Comment by Nitin-Kashyap** [comment]
lgtm

File: presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcPageSourceFactory.java [hunk 1]
```diff
@@ -124,6 +126,7 @@
     public static final String ACID_COLUMN_ROW_ID = "rowId";
     public static final String ACID_COLUMN_CURRENT_TRANSACTION = "currentTransaction";
     public static final String ACID_COLUMN_ROW_STRUCT = "row";
+    public static final Integer SKIP_ORC_FILE_COLUMNS = 7;
```
Comment (sraghunandan) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcPageSourceFactory.java:129]: add comment for what are these columns and their names

**Pull request closed without merging**